MAINTAINERS.md

```diff
@@ -1,9 +1,7 @@
 # Serverless Workflow Org Maintainers
 
-* [Antonio Mendoza Pérez](https://github.com/antmendoza)
 * [Charles d'Avernas](https://github.com/cdavernas)
 * [Ricardo Zanini](https://github.com/ricardozanini)
-* [Tihomir Surdilovic](https://github.com/tsurdilo)
 
```

Comment:
```suggestion
# Serverless Workflow Org Emeritus Maintainers

* [Antonio Mendoza Pérez](https://github.com/antmendoza)
* [Tihomir Surdilovic](https://github.com/tsurdilo)
```

As per our [gov model](https://github.com/serverlessworkflow/specification/blob/main/GOVERNANCE.md#emeritus-maintainers) and their valuable long time contribution.

Also, can you please update the OWNERS [1][2] files? I believe we can stop spamming their inbox with review requests on every PR.

[1] https://github.com/serverlessworkflow/specification/blob/main/owners.md
[2] https://github.com/serverlessworkflow/specification/blob/main/.github/CODEOWNERS

Reply:
Should be fixed. Is everything ok now ?